### PR TITLE
Closes #1419 Registered unregistered models

### LINKED ISSUE #1419: Register admin for remaining models
We haven't registered all project models with the Django Admin, making it difficult to manage content in a centralized manner.

## Task

For each model in our project,
- [ ] register the model with the Django Admin, if not already registered

## Instructions

Start by reading the following guides.

- Django [working with the Django admin](https://docs.djangoproject.com/en/4.1/ref/contrib/admin/) - necessary knowledge to create a pull request with your changes
- GitHub [about working with branches](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches) - necessary knowledge for registering Django models with the Django admin site

The only files that should be changed are admin.py in each app. Look in the models.py for each app and compare it with the admin.py in the same app. Find any models that have not been registered in admin.py and add them.

## Resources

- https://docs.djangoproject.com/en/4.1/ref/contrib/admin/#modeladmin-objects

### PR DESCRIPTION
**Pull request by TimOsahenru**
Closes #1419 

Registered all unregistered models in the admin.py file located inside the account module

### CODE REVIEW

File: project/accounts/admin.py [hunk 1]
```diff
@@ -1,8 +1,29 @@
...
+from threads.models import (
+    Activity,
+    Civi,
+    CiviImage,
+    Fact,
+    Rationale,
+    Rebuttal,
+    Response,
+    Thread,
+)
```
Comment (brylie) [inline project/accounts/admin.py]: Register `threads` models from within the `threads` app in the file `threads/admin.py`.

**Review by brylie — CHANGES_REQUESTED** [review]

File: project/accounts/admin.py (hunk 1 above)
Comment (TimOsahenru) [reply project/accounts/admin.py]: Well noted @brylie. Should I go ahead to also register the Category and Notification model from their respective admin?

Comment (brylie) [reply project/accounts/admin.py]: Each model is registered in its respective admin.py, found within the same application directory.

Comment (TimOsahenru) [reply project/accounts/admin.py]: Oh @brylie I don't think that was the case, only the account module had an admin file that's why I initially registered all models there. However, I have created an admin file for thread, categories and notification modules, can I go ahead to push for a Pull Request?

Comment (brylie) [reply project/accounts/admin.py]: You can commit those changes in your `register-models` branch related to this pull request.

Comment (TimOsahenru) [reply project/accounts/admin.py]: Hello @brylie please I have successfully committed my changes.

**Comment by brylie** [comment]
When creating a pull request that will resolve an issue, please [link the pull request to the related issue(s)](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) by using one of the [supported keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) followed by the issue number in the pull request description.

For example:

```
Closes #<issue-number>
```

## Further details

> You can link a pull request to an issue by using a supported keyword in the pull request's description or in a commit message.
> 
> - close
> - closes
> - closed
> - fix
> - fixes
> - fixed
> - resolve
> - resolves
> - resolved

**Comment by TimOsahenru** [comment]
Hello @brylie thank you for the resources I hope I did it right, I have modified my PR

**Review by brylie — APPROVED** [review]

**Comment by brylie** [comment]
@all-contributors please add @TimOsahenru for code

**Pull request merged**